### ISSUE TITLE
🐞 Coverage report fails on empty project

### ISSUE THREAD

**Issue by ckunki** [exasol/python-toolbox#157]
I used the PTB workflow files for a new project repo saas-python-api which contains no code, yet, and only some placeholders for tests. Hence running tests does not generate any coverage. 

The PTB workflow then [fails](https://github.com/exasol/saas-api-python/actions/runs/8717551503/job/23913034222?pr=9) due to no coverage available:

```
test/integration/test_placeholder.py::test_placeholder PASSED            [100%]

============================== 1 passed in 0.01s ===============================
/home/runner/.cache/pypoetry/virtualenvs/saas-api-dRykLEL3-py3.10/lib/python3.10/site-packages/coverage/control.py:887: CoverageWarning: No data was collected. (no-data-collected)
  self._warn("No data was collected.", slug="no-data-collected")

nox > poetry run coverage report -m
No data to report.
nox > Command poetry run coverage report -m failed with exit code 1
nox > Session coverage failed.
```

## Expected Behaviour

PTB workflows terminate gracefully even on new virgin projects without any test or coverage.

**Comment by ckunki** [comment]
I could fix the CI build by adding an if statement to File `.github/workflows/checks.yml`:
```
      - name: Calculate Test Coverage
        if: ${{ hashFiles('.coverage') != '' }}
        run: poetry run nox -s coverage -- -- --db-version ${{ matrix.exasol-version }}
```

**Comment by Nicoretti** [comment]
This issue does not appear to be a bug within the toolbox itself, but rather a project setup issue. Accordingly, it will be addressed as part of [Issue #166](https://github.com/exasol/python-toolbox/issues/166) on our GitHub repository.
